includes/utilities.inc

```diff
@@ -17,21 +17,38 @@ function islandora_usage_stats_log_object_view(AbstractObject $object) {
     if (($last_viewed + variable_get('islandora_usage_stats_cooldown_seconds', 300)) < REQUEST_TIME) {
       module_load_include('inc', 'islandora_basic_collection', 'includes/utilities');
       module_load_include('inc', 'islandora_usage_stats', 'includes/db');
-      $pid_id = islandora_usage_stats_get_pid_id($object->id);
-      $object_access_id = islandora_usage_stats_object_view_to_db($pid_id);
+      try {
+        $pid_id = islandora_usage_stats_get_pid_id($object->id);
+        $object_access_id = islandora_usage_stats_object_view_to_db($pid_id);
+      }
+      catch (Exception $e) {
+        watchdog('islandora_usage_stats', 'Error: islandora_usage_stats_pid_to_db() could not load %pid', array('%pid' => $object->id), WATCHDOG_ERROR);
```

Comment:
@DiegoPino In a nutshell, the first call to `islandora_usage_stats_pid_to_db()` in `islandora_usage_stats_log_object_view()` is to log the object that got viewed (which can't ever be FALSE or `islandora_usage_stats_log_object_view()` would break first). 

The second call to `islandora_usage_stats_pid_to_db()` is to log the CModel of the object that got viewed, which is pulled directly from the object's RELS-EXT and could, potentially, be invalid.

The third and final call to `islandora_usage_stats_pid_to_db()` is to log parent collections, which are also pulled directly from the RELS-EXT and could potentially point to parent collections that no longer exist.

Of these three calls, only the last two are possible, and only the last one is likely.